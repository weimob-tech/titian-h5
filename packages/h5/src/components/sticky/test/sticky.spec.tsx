import { newSpecPage } from '@stencil/core/testing';
import { h } from '@stencil/core';
import { TiSticky } from '../index';

describe('渲染 ti-sticky', () => {
  it('基本用法', async () => {
    const page = await newSpecPage({
      components: [TiSticky],
      template: () => (
        <ti-sticky>
          <div>吸顶元素</div>
        </ti-sticky>
      ),
    });
    expect(page.root).toMatchSnapshot();
  });
});
