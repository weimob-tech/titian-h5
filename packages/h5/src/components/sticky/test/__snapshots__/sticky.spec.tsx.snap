// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`渲染 ti-sticky 基本用法 1`] = `
<ti-sticky>
  <mock:shadow-root>
    <div class="titian-sticky">
      <div>
        <slot></slot>
      </div>
    </div>
  </mock:shadow-root>
  <div>
    吸顶元素
  </div>
</ti-sticky>
`;
